Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:18.04
+WORKDIR /opt
+
+ARG TAG
+ARG EXTERNAL_IMPORT_BUILD_DIR=/external-import
+
+RUN apt update -qq -y && \
+    apt upgrade -qq -y && \
+    apt install -y \
+    python3 \ 
+    python3-setuptools \
+    python3-pip \
+    wget
+    
+#Install pandas
+RUN pip3 install pandas
+RUN pip3 install testfixtures
+
+RUN ln -s -f /usr/bin/python3 /usr/local/bin/python
+
+# Install external-import
+RUN mkdir -p $EXTERNAL_IMPORT_BUILD_DIR
+COPY . $EXTERNAL_IMPORT_BUILD_DIR
+RUN cd $EXTERNAL_IMPORT_BUILD_DIR \
+    && python3 setup.py clean --all \
+    && python3 setup.py test \
+    && python3 setup.py install \
+    && ln -sf $EXTERNAL_IMPORT_BUILD_DIR/importer /usr/local/bin
```

Comment:
Where does python3 install to? Surprised it doesn't already install to /usr/local/bin